both/routes/userClass.js

```diff
@@ -0,0 +1,22 @@
+Router.map(function () {
+  this.route('userClass/create', {
+    path: '/userClass/create',
+    template: 'userClass'
+  //TODO: Permissions
+  });
+  this.route('userClass/:id/edit', {
```

Comment:
You have a route to edit an individual user class? There should be a single page in which you can manage all the classes. There should also be another route /user/:id/edit in which you can edit add/remove classes from the user.